pages/common/gh-alias.md

```diff
@@ -22,3 +22,7 @@
 - Delete a command shortcut:
 
 `gh alias delete {{alias_name}}`
+
+- Set a shell command to search an open pull request using `fzf` and checkout it:
+
+`gh alias set --shell {{switch}} '{{id="$(gh pr list --limit 1000 | fzf | cut -f1)"; [ -n "$id" ] && gh pr checkout "$id"}}'`
```

Comment:
https://github.com/tldr-pages/tldr/pull/5449/files#r593956566
I'm assuming `--shell` allows you to run the alias as a standalone command e.g. `$ find-pr`
```suggestion
- Add a global shell alias to search for, select, and check out an open pull request using `fzf`:

`gh alias set --shell {{find-pr}} '{{id="$(gh pr list --limit 200 | fzf | cut -f1)"; [ -n "$id" ] && gh pr checkout "$id"}}'`
```
Also don't you think 1000 is a bit much? For example, if you ran this command in the [kubernetes/kubernetes](https://github.com/kubernetes/kubernetes) repo it might take a _**while**_....

Reply:
It's a quite confusing `for, select, and check out`.
Maybe 1000 is really big.

Reply:
It's really just a more concise version of this (the words in bold are implied but not actually written):

>Add a global shell alias to search for **an open pull request**, **then** select **the open pull request**, and **finally** checkout the open pull request.
 
---

About `--shell` — from the [docs](https://cli.github.com/manual/gh_alias_set):

>If '--shell' is specified, the alias will be run through a shell interpreter (sh). This allows you to compose commands with "|" or redirect with ">". 

So @navarroaxel how about this?
- Add a global shell alias to search for, select, and check out an open pull request using `fzf`:

```suggestion
- Add an alias to search for, select, and check out an open pull request using `fzf` (`--shell` allows for shell syntax inside the alias):

`gh alias set --shell {{find-pr}} '{{id="$(gh pr list --limit 200 | fzf | cut -f1)"; [ -n "$id" ] && gh pr checkout "$id"}}'`
```
cc @sbrl

Reply:
@bl-ue `--shell` is explained in another example above this.

Reply:
Oh oops, I didn't see that.  Suggestion updated.